pom.xml

```diff
@@ -293,7 +293,7 @@
         <version>2.2</version>
         <configuration>
           <path>/</path>
-          <contextFile>/etc/ambra/context.xml</contextFile>
+          <contextFile>${ENV.home}/ambra/config/context.xml</contextFile>
```

Comment:
Just a thought. On the one hand YAGNI but on the other would be consistent with the corresponding PR's for rhino and wombat if you added a contentRepo.configDir element and referenced it here.

Reply:
I tried that first, but then I figured YAGNI